native-maven-plugin/README.md

```diff
@@ -6,10 +6,12 @@ End-user documentation about the plugins can be found [here](https://graalvm.git
 
 ## Building
 
-This plugin follows standard Maven plugin conventions and can be built with this command:
+This plugin can be built with this command (from the root directory):
 
-```shell
-./gradlew publishAllPublicationsToCommonRepository
+```bash
+./gradlew :native-maven-plugin:publishToMavenLocal --no-parallel
```

Comment:
Have you tried this command if it works?

Reply:
Yes it will work. Although personally I'd avoid polluting maven local and stick with the "common repository" (I usually add a local repository to Gradle/Maven builds to avoid the erratic behavior of maven local).

Reply:
@melix could you explain how to use the common repository? After grepping the repository I still don't really understand how it works.

I tried `./gradlew publishAllPublicationsToCommonRepository --no-parallel` but my changes were not picked up 
(presumably I need to use it in my project's build definition).

Reply:
There is nothing special here, `publishAllPublicationsToCommonRepository` will update the repository found in `build/common-repo`, and for sure you need to use it in your project definition. For example in Gradle you'd simply add a repository which points to `build/common-repo`, and in Maven you'd add a `<repository>`. You don't have to go that way, you can publish to Maven local, but I personally think it's a bad practice, because Maven local is polluted with dozens of libraries which may make us accidentally think that a build is working, when if you try it on a different machine, it would break.